chrysom/metrics.go

```diff
@@ -3,12 +3,6 @@
 
 package chrysom
 
-import (
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/xmidt-org/touchstone"
-	"go.uber.org/fx"
-)
-
 // Names
 const (
```

Comment:
is it necessary to have this file just for three constants? would it make sense to move them to another file?

Reply:
I think so. I may even move the touchstone options in here